src/utils/pause.js

```diff
@@ -1,7 +1,10 @@
+// @flow
 const { Frame } = require("../tcomb-types");
 const { getOriginalLocation } = require("./source-map");
 
-function updateFrameLocations(frames) {
+import type { Frame as TFrame } from "../types";
+
+function updateFrameLocations(frames: TFrame[]): Promise<TFrame[]> {
```

Comment:
small nit, when there's a conflict we use `FrameType`

https://github.com/montogeek/debugger.html/blob/flow-type-pause-actions-reducer/src/actions/sources.js#L273-L275